.github/workflows/developer_onboarding_notification.yml

```diff
@@ -0,0 +1,64 @@
+
+name: PR Merge Comment
+
+on:
+  pull_request_target:
+    types: [closed]
+
+permissions: write-all
+
+jobs:
+  comment-on-merge:
+    if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'develop'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/merge
```

Comment:
What does this "step" do? Should it have a name?

Reply:
If I don't include this then get this lint error:
![image](https://github.com/user-attachments/assets/e266b0c6-5211-4f30-8356-2e969b7d5ad7)